File: week3/이다혜/음양더하기.java
public class 음양더하기 {
    
    public int solution(int[] absolutes, boolean[] signs) {
        int answer = 0;
        int n = absolutes.length;

        for(int i = 0; i < n; i++) {
            if(signs[i] == true) {
                answer += absolutes[i];
            } else {
                answer -= absolutes[i];
            }
        }
        return answer;
    }

}

File: week3/윤여준/음양 더하기.cpp
#include <bits/stdc++.h>
using namespace std;

int solution(vector<int> absolutes, vector<bool> signs) {
    int answer = 0;
    
    for(int i=0;i<absolutes.size();i++) {
        if(!signs[i]) absolutes[i] = -absolutes[i];
        answer += absolutes[i];
    }
    
    return answer;
}